algosdk/future/transaction.py

```diff
@@ -1650,23 +1651,24 @@ def teal_bytes(teal):
         ), "Program {} is not bytes".format(teal)
         return teal
 
+    @staticmethod
+    def as_bytes(e):
+        """Confirm or coerce element to bytes."""
+        if isinstance(e, (bytes, bytearray)):
+            return e
+        if isinstance(e, str):
+            return e.encode()
+        if isinstance(e, int):
+            # Uses 8 bytes, big endian to match TEAL's btoi
+            return e.to_bytes(8, "big")  # raises for negative or too big
+        assert False, "{} is not bytes, str, or int".format(e)
```

Comment:
Is there a more informative error that can be raised as apposed to an `AssertionError` ?

Reply:
How about `TypeError` with that message?